### ISSUE TITLE
Add HTML to accordions on record pages

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-frontend#394]
**Problem Description:**
Google Analytics crawlers only crawl links available when the page loads. Since accordion links appear upon selection, not on page load, crawlers ignore them.

**Expected Behavior/Solution:**
The list of URIs contained in each accordion, respectively, will be exposed in the rendered HTML so the crawler can read them.

**Requirements:**
- The list of URIs will be contained within their corresponding accordion on page load
- Accordion functionality will remain the same to everyday users

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [x] ~Wireframe/Mockup - Mike~
- [x] ~Committee discussions - Sarah~
- [x] ~Feasibility/Team discussion - Sarah~
- [x] ~Backend requirements - TBD~
- [x] Frontend requirements - Kam
- [ ] Are new regression tests required for QA - Amy
- [x] ~Questions~

**UAT/LUX Examples:**
N/A

**Dependencies/Blocks:**
N/A

**Related Github Issues:**
N/A

**Related links:**
N/A

**Wireframe/Mockup:**
Front end remains the same to users.

**Comment by roamye** [comment]
@kamerynB - would a UAT example be to check more tools -> developer on the record page?

**Comment by kamerynB** [comment]
The current functionality is that the frontend retrieves the HAL link results on page load. Once the user opens the accordions, the API requests are made for each individual result. Is the intent that the frontend will make all of those individual result requests on page load as well? @roamye 

cc: @clarkepeterf

**Comment by roamye** [comment]
@kamerynB - 

I thought adding the HTML to the accordions would do allow for the information in the accordions to load upon the page load. If this is not correct, what is the intention behind adding the HTML to the accordions?

**Comment by clarkepeterf** [comment]
@kamerynB When the page loads, it gets all of the related list results, so we already have the information needed to link to each entity, it just won't have a nice link name.

When someone opens up the accordion, we do a `useGetItemQuery()` to get the name of the entity for the link.

Before the accordions are opened, is it possible to include links that are hidden and are just named after the URI instead of the entity name?  That way, google will be able to crawl the related lists.

**Comment by azaroth42** [comment]
Issue should be updated with the info from @clarkepeterf above

**Comment by kamerynB** [comment]
@clarkepeterf So I'm clear, the frontend would not be making the requests to each individual entity, it would just be populating a collapsed accordion with the URIs of those entities?

**Comment by clarkepeterf** [comment]
@kamerynB correct. It should populate the collapsed accordion with links to those uris so that google can crawl them.

**Comment by kamerynB** [comment]
@clarkepeterf Thank you. That makes sense.

**Comment by roamye** [comment]
Closing as it was moved to PROD. 

Note - @prowns and I should compare the analytics from next week to the previous week and monitor any changes.
